### PR TITLE
Add text for Learn section

### PR DESCRIPTION
**Pull request by stargaser**
Add text for Learn section.

### CODE REVIEW

File: src/ms.tex [hunk 1]
```diff
@@ -303,13 +303,54 @@ \section{Learn Astropy} \label{sec:learn}
...
+in user engagement and in conducting workshops. The Learn team meets
+weekly to drive these issues forward.
+
 \begin{itemize}
-\item Current status and scope
-\item Vision for the future (review what we said in v2.0 paper)
-\item User forums and engagement with user base
-\item Workshops
+
+\item {\bf Learn Astropy website:} The site has a new search interface powered by
```
Comment (eblur) [inline src/ms.tex:317]: Jonathan Sick will add a paragraph here.

File: src/ms.tex [hunk 2]
```diff
@@ -303,13 +303,54 @@ \section{Learn Astropy} \label{sec:learn}
...
+supported in part by the Dunlap grant. Tutorials have been added and
+a {\it ccdproc} Jupyter book has been completed.
+
+\item {\bf User forums:} In addition to the pre-existing Astropy mailing list,
+and the Python in Astronomy Facebook group, users have been invited to the
+Astropy Slack workspace. The Moore Foundation grant allows this Slack space to
+be on a paid plan; additionally NumFOCUS has negotiated a special rate for
+open-source projets. There is some internal debate amongst Astropy developers
+on whether Astropy Slack should be reserved for developer discussions, with
+users redirected to another site. Accordingly, the Astropy Project has commissioned
```
Comment (eblur) [inline src/ms.tex:328]:
```suggestion
\item {\bf User forums:} The Astropy Project has historically maintained a number of avenues for users to seek help or gain access to the developer community. This includes the Astropy mailing list, the Python in Astronomy Facebook group, and the Astropy Slack workspace. The Moore Foundation grant allows this Slack space to
be on a paid plan; additionally NumFOCUS has negotiated a special rate for
open-source projets. While the Slack and Astropy-dev mailing lists are primarily used to discuss the project direction and updates, it was noted that the use of a Facebook community could present a barrier to open source. The Astropy Project identified a need for a public, archived, searchable, and easily-to-navigate interface for users to ask for help Accordingly, we have commissioned 
```

File: src/ms.tex [hunk 3]
```diff
@@ -303,13 +303,54 @@ \section{Learn Astropy} \label{sec:learn}
...
+\item {\bf Workshops:} the Project has been conducting workshops at winter meetings
+of the American Astronomical Society since AAS 225 in January 2015. Up to the
+start of the coronavirus pandemic, these were full-day in-person workshops with
+as many as 90 participants and a dozen facilitators from the project. During the
+pandemic, these workshops were moved to an online format and split into
+basic and advanced sessions. Additionally, beginning with the AAS 238 online
+meeting, the workshops have been expanded to Summer AAS meetings. The Learn
+team finds that the workshop audience is best found as AAS meetings as opposed
+to more general Python meetings.
+
```
Comment (eblur) [inline src/ms.tex:343]:
```suggestion

The Astropy Project provided another mode of community engagement at AAS Meetings 235 and 237 by organizing a NumFOCUS Sponsored Projects booth in the AAS Exhibit Hall. Funding for the exhibit hall was provided alternately by NumFOCUS and later by the Moore Foundation funding (\textbf{check this}). The booth hosted a series of Q\&A special sessions during AAS 235 and webinars during the virtual AAS 237 meetings, to provide the general astronomy community information and access to experts on a variety of open source astronomical tools.
```

File: src/ms.tex [hunk 4]
```diff
@@ -303,13 +303,54 @@ \section{Learn Astropy} \label{sec:learn}
 
 \secauthor{Lía Corrales, David Shupe + Learn team}
 
+\subsubsection{Current status and scope}
+
+The Astropy-ii paper discussed the {\it Learn Astropy} effort only in the
+context of online tutorials and guides. This vision has been realized as
+{\it Learn Astropy} has released a new version of this website, and has
+expanded its efforts
```
Comment (eblur) [inline src/ms.tex:311]:
```suggestion
{\it Learn Astropy} is an umbrella term that acknowledges the broad educational efforts made by the Astropy Project, which are led by the Astropy Learn Team. 
As described in \textsl{[Astropy-ii paper]}, Learn Astropy content has been categorized along four axes: documentation, which is contained within the python code base; tutorials, consisting of Jupyter Notebook lessons that are published in HTML format online; guides, which are a series of lessons providing a foundational resource for performing a certain type of astronomical analyses; and examples, which are snippets of code that showcase a short task that can be performed with Astropy or its affiliated package. 
These categories continue to drive both the infrastructure choices and the appearance of the Learn Astropy website. Additionally, the Astropy Learn Team has expanded its efforts
```

File: src/ms.tex (hunk 1 above)
Comment (stargaser) [reply src/ms.tex:316]: @jonathansick here's the spot needing your inputs

File: src/ms.tex [hunk 5]
```diff
@@ -301,18 +301,59 @@ \subsection{Roadmap}
...
 
 \begin{itemize}
-\item Current status and scope
-\item Vision for the future (review what we said in v2.0 paper)
-\item User forums and engagement with user base
-\item Workshops
+
+\item {\bf Learn Astropy website:} The website has a new search interface powered by
+Algolia which is used by major e-commerce websites. This work has been
+supported in part by the Dunlap grant.
```
Comment (jonathansick) [inline src/ms.tex:318]: How about something like this?

```suggestion
\item {\bf Learn Astropy website:} The Learn Team re-launched the website in 2021 with a new infrastructure platform, built around full-text search and interactive filtering functionality, with the goal of making content discoverable as the Learn Astropy content catalog expands.
This work has been
supported in part by the Dunlap grant.
We have adopted Algolia, a search-as-a-service cloud platform, to store the full-text and metadata records of Learn Astropy's content.
The new Learn Astropy website is a JavaScript (Gatsby/React) application that uses the Algolia service to power its search and filtering user interface.
Our Python-based application, Learn Astropy Librarian, populates data into the Algolia service. We tuned the Librarian around specific content formats (such as Jupyter Notebook-based tutorial pages and Jupyter Book-based guides) to more accurately index content and heuristically extract metadata.
A consequence of the new platform is that we now maintain and compile content separately from the website application itself, enabling new content types.
Tutorials, which as Jupyter Notebook-based, are now compiled into their own Learn Astropy sub-site using nbcollection.
Guides, which are Jupyter Book-based, are also deployed as separate websites using GitHub pages. This architecture opens future possibilities of indexing third-party content, hosted elsewhere, such as on institutional websites.
```

Comment (stargaser) [reply src/ms.tex:318]: I think this is good and I'll commit the suggestion. We probably want to describe "Dunlap grant".

File: src/ms.tex (hunk 1 above)
Comment (stargaser) [reply src/ms.tex]: suggested text added.

File: src/ms.tex [hunk 6]
```diff
@@ -301,18 +301,64 @@ \subsection{Roadmap}
...
 \section{Learn Astropy} \label{sec:learn}
 
-\secauthor{Lía Corrales, David Shupe + Learn team}
+\secauthor{Lía Corrales, David Shupe, Kelle Cruz + Learn team}
+
+\subsubsection{Current status and scope}
+
+{\it Learn Astropy} is an umbrella term that acknowledges the broad educational efforts made by the Astropy Project, which are led by the Learn Team.
+The efforts focus on developing online content and workshops covering astro-specific coding tasks in Python.
+As described in \textsl{[Astropy-ii paper]}, There are four different types of Learn Astropy content: \textit{tutorials}, consisting of Jupyter Notebook lessons that are published in HTML format online; \textit{guides}, which are a series of lessons providing a foundational resource for performing a certain type of astronomical analyses; \textit{examples}, which are snippets of code that showcase a short task that can be performed with Astropy or an affiliated package; and \textit{documentation}, which is contained within the code base.
```
Comment (nmearl) [inline src/ms.tex:310]:
```suggestion
As described in \citep{astropy:2018}, there are four different types of Learn Astropy content: \textit{tutorials}, consisting of Jupyter Notebook lessons that are published in HTML format online; \textit{guides}, which are a series of lessons providing a foundational resource for performing certain type of astronomical analyses; \textit{examples}, which are snippets of code that showcase a short task that can be performed with Astropy or an affiliated package; and \textit{documentation}, which is contained within the code base.
```

**Review by nmearl — CHANGES_REQUESTED** [review]

File: src/ms.tex [hunk 7]
```diff
@@ -301,18 +301,98 @@ \subsection{Roadmap}
...
-\secauthor{Lía Corrales, David Shupe + Learn team}
+\secauthor{Lía Corrales, David Shupe, Kelle Cruz + Learn team}
+
+\subsubsection{Current status and scope}
+
+{\it Learn Astropy} is an umbrella term that acknowledges the broad educational
+efforts made by the Astropy Project, which are led by the Learn Team.
+The efforts focus on developing online content and workshops covering
+astro-specific coding tasks in Python.
+As described in \textsl{[Astropy-ii paper]}, There are four different types of
```
Comment (nmearl) [inline src/ms.tex:312]:
```suggestion
As described in \citep{astropy:2018}, There are four different types of
```

File: src/ms.tex [hunk 8]
```diff
@@ -301,18 +301,98 @@ \subsection{Roadmap}
...
+Learn Astropy content: \textit{tutorials}, consisting of Jupyter Notebook
+lessons that are published in HTML format online; \textit{guides}, which are a
+series of lessons providing a foundational resource for performing a certain
```
Comment (nmearl) [inline src/ms.tex:315]:
```suggestion
series of lessons providing a foundational resource for performing certain
```

**Review by nmearl — CHANGES_REQUESTED** [review]

**Review by adrn — APPROVED** [review]
Thanks @stargaser - I'm going to merge this and work on adding text about the tutorials and guides in a separate PR.

**Pull request merged**